### ISSUE TITLE
Solution in-world so new users know what controls they have

### ISSUE THREAD

**Issue by DougReeder** [Hubs-Foundation/hubs#6526]
**Is your feature request related to a problem? Please describe.**
Many, if not most, rooms need to make new users feel welcome. (See https://web.archive.org/web/20240520035510/https://hubs.mozilla.com/labs/setting-the-scene-immersive-space-design/ for why)
One issue is most users are new to Hubs, and don't know the controls
The examples in the blog post have posters or videos in-world to help users learn the controls.  However, these controls are different for different platforms, and we currently have no good way to make content responsive to user platform.

Also, the easier it is for world creators to include this, the more worlds will have it, and the more users will have positive first experiences with Hubs.

**Describe the solution you'd like**
Ideally, we'd have a general solution for elements that are responsive to the user platform.  We would provide standard text & diagrams for world builders to put in.

**Describe alternatives you've considered**
We have a UI element on personal computers that provides a few tips, but this is not available on other platforms and provide very little content.

We could generate graphics with the controls for each platform. These could be placed in Image elements. However users that first see content irrelevant to them in one image element are less likely to look at the others. Also, a "wall of text" is not a welcoming experience.

A single-purpose element that just gave directions would work, but would provide less value.

As a hack, I include a link to https://github.com/Hubs-Foundation/hubs/wiki/Hubs-Controls in my worlds near the spawn point, but that takes users out of the world.

**Additional context**
[Virtual Ability Island in Second Life](https://web.archive.org/web/20240520035510im_/https://hubs.mozilla.com/labs/content/images/2023/06/Blank-PowerPoint-Presentation.png)

[NYU VR Orientation Space](https://web.archive.org/web/20240520035510im_/https://hubs.mozilla.com/labs/content/images/2023/06/Blank-PowerPoint-Presentation-2-.png)

**Comment by Exairnous** [comment]
I agree that the new user experience should be improved, but I don't think that's the job of world builders (although I do agree that a general solution for elements that are responsive to the user platform would be a good addition; this would probably take the form of a Behavior Graph node to detect which device the user is using).

I think instead that the little tutorial for new users should be expanded to be more complete and cross-platform.  We could also include links at the end to more information and/or an official training room if people want to practice (although potentially the training room link should be an admin configurable link and the Hubs Foundation just provides a scene that an admin can import if they want).

I actually have some old files for a tutorial scene that I never got finished, but could provide if anyone wants (others in the community have similar stuff, like the NYU VR Orientation Space you mentioned which is by Innovate).  If we go this route, we should probably make sure the scene is designed in such a way that it can be easily updated for new features.

**Related tasks:**
https://github.com/Hubs-Foundation/hubs/issues/1992
https://github.com/Hubs-Foundation/hubs/issues/3128
https://github.com/Hubs-Foundation/hubs/issues/1659

**Note:** This looks like a duplicate of https://github.com/Hubs-Foundation/hubs/issues/1992, so we should probably continue the discussion there.

**Comment by DougReeder** [comment]
These are all facets of the same underlying issue. It would help to have a meta-issue summarizing and pointing to all of them.
